_posts/2019-05-08-model-serving-in-pyorch.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: blog_detail
+title: 'Model Serving in PyTorch'
+author: Jeff Smith
+redirect_from: /2019/05/08/model-serving-in-pyorch.html
+---
+
+# Model Serving in PyTorch
+
+PyTorch has a deservedly awesome reputation for use in research, but people can get confused about how well PyTorch models can be taken into production. This blog post is meant to clear up any confusion people might have about the road to production in PyTorch.
```

Comment:
" has a deservedly awesome reputation"

We're authoring the blogpost as the PyTorch team, i think we shouldn't use superlatives about ourselves

Reply:
Edited to be more modest.